server/openejb-cxf/pom.xml

```diff
@@ -191,7 +195,7 @@
           <artifactId>jaxb-xjc</artifactId>
         </exclusion>
         <exclusion>
-          <groupId>net.sf.ehcache</groupId>
+          <groupId>org.ehcache</groupId>
```

Comment:
Good catch. Seems they upgrade `ehcache` from v2 to v3 with 3.4.x

Don't know, if this affects anything in TomEE itself as we are still have `ehcache` v2 in the classpath. Consequently, I am wondering, if this will be an problem, if we exclude the v3 `ehcache` here`. 

Might be also relevant for other libs, which come with v3 (such as `wss4j-ws-security-stax` - exclude on `ǹet.sf` doesnt work anymore...). Consequently, I am wondering, if we need to upgrade the `ehcache` version in TomEE as well (would require to add  `org.ehcache` to the exclusion lists in `ContainerClassesFilter` and `JuliLogStreamFactory`).

Maybe @dblevins have some thoughts on it?

Reply:
I did this looking at the lib folder in the generated tomee, didn't see the old ehcache there, but yes, maybe it's better to also upgrade as well, don't really know for sure.

Reply:
Might be interesting to do some `mvn dependency:tree` analysis related to ehcache? :)

Reply:
Agree that if CXF upgraded from `ehcache` v2 to v3, we should do the work to ensure that's the one we're using in the pomx and update our exclusion lists as/if applicable for the new groupId.

Reply:
It seems, that the V2 dependency (defined in the root pom) is only used in `openejb-core-hibernate` module, which is correct as `hibernate-ehcache` is still using V2, see [dependency:tree](https://gist.github.com/rzo1/86d87a50bce0e1f8c41f87edc0618a17).

Nevertheless, it might be worth adding `org.ehcache` to the exclusion lists in `ContainerClassesFilter` and `JuliLogStreamFactory` (to be future safe...). Besides that, we are probably good here.